apps/parsley/src/types/logs.ts

```diff
@@ -3,7 +3,20 @@ import { CaseSensitivity, MatchType } from "constants/enums";
 type ExpandedLine = [number, number];
 type ExpandedLines = ExpandedLine[];
 
-type ProcessedLogLine = number | number[];
+interface Range {
+  /** The starting line inclusive of the range */
+  lineStart: number;
+  /** The ending line exclusive of the range */
+  lineEnd: number;
+}
```

Comment:
I was curious about the name here, about if they need to be called `lineStart` and `lineEnd` as opposed to `start` and `end`. `lineStart` and `lineEnd` are from the technical design but maybe these names aren't necessary given the introduction of a new object